### ISSUE TITLE
Should we show message for unversioned artifacts in Info palen

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-template-editor#810]
Currently Version tab is blank in Info panel for an unversioned (0.0.1/draft) artifact.

Perhaps have "This item is not yet versioned" message?

**Comment by graybeal** [comment]
yes please (looking for 'vote' widget). Also include a link to our documentation on versioning. I don't want anyone versioning something if they don't know the implications.
